### DISCUSSION TITLE
Auto-search for radio stations with auto-saving them into memory cells

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#308] [Ideas]
Is it possible to add the ability to auto-search with auto-saving radio stations to memory cells? As an implementation option, we turn on the desired range, go to the main settings menu, there we launch the Autosave stations item and the receiver scans the entire current range and writes radio stations to memory cells

**Comment by Yurij20** [comment]
Is it possible to implement my proposal?

**Comment by PE5PVB** [comment]
Maybe, but for future releases. I'm now finalising this version for release.

**Reply by Yurij20** [reply]
> Maybe, but for future releases. I'm now finalising this version for release.

I’ve written about this idea before and I hasten to remind you about it, because I think it’s important for traveling in order to more conveniently study the frequency plan of the city you’re visiting. To do this, you need to enter: 1. Auto-search for radio stations of the current range with auto-saving them into memory. 2. Auto-sorting of memory channels in order of increasing frequency with their compaction and filling of deleted memory cells as a result of removing phantom frequencies after auto-scanning. 3. Completely delete all saved stations from memory. If you start the auto search without clearing all the memory, the auto search fills the remaining free memory cells. This functionality is perfectly implemented in one of the modifications of firmware 1.15. I can attach a link if necessary.

**Comment by PE5PVB** [comment]
Well.. it's there now.
